extensions/default/src/Panels/PanelStudyBrowser.tsx

```diff
@@ -123,24 +123,28 @@ function PanelStudyBrowser({
   useEffect(() => {
     const currentDisplaySets = displaySetService.activeDisplaySets;
     currentDisplaySets.forEach(async dSet => {
-      const newImageSrcEntry = {};
       const displaySet = displaySetService.getDisplaySetByUID(
         dSet.displaySetInstanceUID
       );
       const imageIds = dataSource.getImageIdsForDisplaySet(displaySet);
       const imageId = imageIds[Math.floor(imageIds.length / 2)];
 
       // TODO: Is it okay that imageIds are not returned here for SR displaySets?
-      if (imageId && !displaySet?.unsupported) {
+      if (
```

Comment:
A better choice is to configure OHIF to use the server side rendered thumbnails.  Those only need series instance UID, so it should be possible to just render those as simple requests.

Reply:
I doens appear to be supported abroad. I tried /studies/{study}/series/{series}/thumbnail  with Orthanc and it didnt work.